homeassistant/components/edl21/translations/en.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Please remove the translation file. Following recent changes it should no longer be added to PRs

Reply:
Ok, I removed the translation file.